tests/integration/targets/aci_access_spine_interface_profile/tasks/main.yml

```diff
@@ -0,0 +1,206 @@
+# Test code for the ACI modules
+# Copyright: (c) 2020, Shreyas Srish <ssrish@cisco.com>
+# Copyright: (c) 2017, Bruno Calogero <brunocalogero@hotmail.com>
+# Adapted from aci_interface_policy_leaf_profile tests
+# Copyright: (c) 2023, Eric Girard <@netgirard>
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: Test that we have an ACI APIC host, ACI username and ACI password
+  fail:
+    msg: 'Please define the following variables: aci_hostname, aci_username and aci_password.'
+  when: aci_hostname is not defined or aci_username is not defined or aci_password is not defined
+
+- name: Set vars
+  set_fact: 
+   aci_info: &aci_info
+    host: "{{ aci_hostname }}"
+    username: "{{ aci_username }}"
+    password: "{{ aci_password }}"
+    validate_certs: "{{ aci_validate_certs | default(false) }}"
+    use_ssl: "{{ aci_use_ssl | default(true) }}"
+    use_proxy: "{{ aci_use_proxy | default(true) }}"
+    output_level: "{{ aci_output_level | default('info') }}"
+
+- name: Verify Cloud and Non-Cloud Sites in use.
+  include_tasks: ../../../../../../integration/targets/aci_cloud_provider/tasks/main.yml
+
+- name: Execute tasks only for non-cloud sites
+  when: query_cloud.current == []  # This condition will execute only non-cloud sites
+  block:  # block specifies execution of tasks within, based on conditions
+  # CLEAN ENVIRONMENT
+  - name: Remove spine profile
+    cisco.aci.aci_access_spine_interface_profile: &interface_policy_spine_profile_absent
+      <<: *aci_info
+      spine_interface_profile: ansible_test
+      state: absent
+
+  # ADD SPINE INTERFACE PROFILE
+  - name: Add spine interface profile (check_mode)
+    cisco.aci.aci_access_spine_interface_profile: &interface_policy_spine_profile_present
+      <<: *aci_info
+      interface_profile: ansible_test
+      state: present
+    check_mode: yes
+    register: cm_add_spine_interface_profile
+
+  - name: Add spine interface profile (normal mode)
+    cisco.aci.aci_access_spine_interface_profile: *interface_policy_spine_profile_present
+    register: nm_add_spine_interface_profile
+
+  - name: Add spine interface profile again (check_mode)
+    cisco.aci.aci_access_spine_interface_profile: *interface_policy_spine_profile_present
+    check_mode: yes
+    register: cm_add_spine_interface_profile_again
+
+  - name: Add spine interface profile again (normal mode)
+    cisco.aci.aci_access_spine_interface_profile: *interface_policy_spine_profile_present
+    register: nm_add_spine_interface_profile_again
+
+  - name: Verify add_spine_interface_profile
+    assert:
+      that:
+      - cm_add_spine_interface_profile is changed
```

Comment:
Are there missing asserts here?
Like proposed and other attributes? Could assert dn.